### PR TITLE
Fix/unusedremove

### PR DESCRIPTION
**Pull request by NikAshanin**

### CODE REVIEW

File: LeadKit/LeadKit/Extensions/CGContext/CGContext+Initializers.swift [hunk 1]
```diff
@@ -1,33 +1,46 @@
...
-                              fallbackColorSpace: CGColorSpace = CGColorSpaceCreateDeviceRGB()) -> CGContext? {
-
-        return create(width: cgImage.width,
-                      height: cgImage.height,
-                      bitmapInfo: cgImage.bitmapInfo,
-                      colorSpace: cgImage.colorSpace ?? fallbackColorSpace,
-                      bitsPerComponent: cgImage.bitsPerComponent)
+    public convenience init?(forCGImage cgImage: CGImage,
+                             fallbackColorSpace: CGColorSpace = CGColorSpaceCreateDeviceRGB()) {
+        self.init(width: cgImage.width,
```
Comment (fedorsemenchenko) [inline LeadKit/LeadKit/Extensions/CGContext/CGContext+Initializers.swift]: is it possible don't use `init` word `self(width`?

File: LeadKit/LeadKit/Extensions/CGContext/CGContext+Initializers.swift [hunk 2]
```diff
@@ -37,19 +50,18 @@ public extension CGContext {
...
-                         bitsPerComponent: bitsPerComponent,
-                         bytesPerRow: 0,
-                         space: colorSpace,
-                         bitmapInfo: bitmapInfo.rawValue)
+    public convenience init?(width: Int,
+                     height: Int,
+                     bitmapInfo: CGBitmapInfo = alphaBitmapInfo,
+                     colorSpace: CGColorSpace = CGColorSpaceCreateDeviceRGB(),
+                     bitsPerComponent: Int = 8) {
+        self.init(data: nil,
```
Comment (fedorsemenchenko) [inline LeadKit/LeadKit/Extensions/CGContext/CGContext+Initializers.swift]: same?

File: LeadKit/LeadKit/Extensions/CGContext/CGContext+Initializers.swift (hunk 1 above)
Comment (NikAshanin) [inline LeadKit/LeadKit/Extensions/CGContext/CGContext+Initializers.swift]: https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Initialization.html
can't be omitted in convenience init =(

File: LeadKit/LeadKit/Extensions/CGContext/CGContext+Initializers.swift (hunk 2 above)
Comment (NikAshanin) [inline LeadKit/LeadKit/Extensions/CGContext/CGContext+Initializers.swift]: same

**Pull request merged**